chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,7 +150,10 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
-            d.fixed and not d.claimed?
+            # skip claimed disks and multipath holders
```

Comment:
IIUC it's not multipath *holders* which are being skipped here, it's devices *held* by holders.